luthier/src/main/java/com/yahoo/bard/webservice/data/config/LuthierApiMetricName.java

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.data.config;
+
+import com.yahoo.bard.webservice.data.config.names.ApiMetricName;
+import com.yahoo.bard.webservice.data.time.TimeGrain;
+
+/**
+ * Holds a metric name which is stored in fili.
+ */
+public class LuthierApiMetricName implements ApiMetricName {
+    private final String apiName;
+    private final TimeGrain timeGrain;
+
+    /**
+     * Constructs a LuthierApiMetricName.
+     *
+     * @param name  The name of the metric.
+     * @param timeGrain  A valid timeGrain for this metric.
+     */
+    public LuthierApiMetricName(String name, TimeGrain timeGrain) {
+        this.apiName = name;
+        this.timeGrain = timeGrain;
+    }
+
+    @Override
+    public String asName() {
+        return getApiName();
+    }
+
+    @Override
+    public String toString() {
+        return apiName;
+    }
+
+    @Override
+    public boolean isValidFor(TimeGrain grain) {
+        return grain.satisfiedBy(timeGrain);
+    }
+
+    @Override
+    public String getApiName() {
+        return toString();
+    }
+}
```

Comment:
We should drop this in favor of FiliApiMetricName until we need a different version.

Reply:
done.